themes/bootstrap3/js/truncate.js

```diff
@@ -76,7 +76,7 @@ VuFind.register('truncate', function Truncate() {
           rowHeight = parseFloat(container.css('line-height').replace('px', ''));
         }
         numRows = Math.ceil(container.height() / rowHeight);
-        shouldTruncate = rowCount < numRows;
+        shouldTruncate = rowCount + 1 < numRows;
```

Comment:
What is the motivation for this change? Is the idea that if there's only one extra row, it's silly to hide it behind a button because the button takes up as much space as the row itself? While that makes sense to me, it's also potentially confusing, because now if I configure VuFind to show a toggle control if there are more than 3 subjects, I don't see the button on records with 4 subjects, which could make me think that something is broken. I think we either need to document the behavior more clearly, or we need a separate setting to control the "fuzziness" of the control (since perhaps in some situations, we want to have a larger threshold over the limit before we actually display the control; I could imagine using a value of 2 or 3 here in some cases).

Reply:
The motivation was that when it truncates by height I was seeing situations where it was only truncating off like half of the last item. If it's confusing, we can undo it.

Reply:
Hmm, I wonder if the solution to that problem is a correction to the formula that figures out the number of rows, rather than changing the requested number of rows. Do you have a consistent way to reproduce the problem? Maybe there's some problem with margins, padding, etc., not being fully accounted for in the calculations.